src/lightning/fabric/loggers/csv_logs.py

```diff
@@ -91,11 +91,6 @@ def version(self) -> Union[int, str]:
             self._version = self._get_next_version()
         return self._version
 
-    @property
-    def root_dir(self) -> str:
-        """Gets the save directory where the versioned CSV experiments are saved."""
-        return self._root_dir
```

Comment:
Can you describe the difference in how PL and Fabric treat the `root_dir`?

And if this is irreconcilable, should `Fabric.root_dir` raise an exception to avoid silent errors (with a comment describing why)? Because the base interface does provide a default implementation which would not be right to call

Reply:
In PL, `root_dir` is misleading since it doesn't point to the root, but rather to `os.path.join(self.save_dir, self.name)`. In Fabric, root_dir is what save_dir is in PL, giving its name an actual meaningful purpose.

> And if this is irreconcilable, should Fabric.root_dir raise an exception to avoid silent errors (with a comment describing why)?

You mean making `CSVLogger.root_dir`in Fabric raise an exception?

Reply:
> You mean making CSVLogger.root_dirin Fabric raise an exception?

Yes, sorry

Reply:
Sorry, my bad. The removal is not necessary. Resolving